src/client/Client.cc

```diff
@@ -9384,7 +9391,8 @@ int Client::_rmdir(Inode *dir, const char *name, int uid, int gid)
     goto fail;
   if (req->get_op() == CEPH_MDS_OP_RMDIR) {
     req->set_dentry(de);
-    req->set_inode(in);
+    req->set_inode(dir);
+    req->set_other_inode(in);
```

Comment:
Why are we swapping inode with other_inode? We're not using inode in put_request()